client/html/layouts/application.html

```diff
@@ -14,7 +14,8 @@
         src_url      : "http://www.openassessments.com/api/assessments/55.xml",
         assessment_kind: "SUMMATIVE",
         questions_per_section:1,
-        questions_per_page:1
+        questions_per_page:1,
+        unlock_next: "ON_CORRECT" 
```

Comment:
Added setting to determine when user should be allowed to move on to the next set of questions